### PR TITLE
ADD: Ionicons 5

### LINKED ISSUE #294: Upgrade to IonIcons v5
Hi,

Thanks for making this super useful library (and the website as well :) ). I was just wondering if we could upgrade this to IonIcons v5 since it just came out and has many new good-looking icons over there. Let me know if I could help in some way.

### PR DESCRIPTION
**Pull request by florianstahr**
This PR adds the Ionicons v5. Closes #294 & #333. Thanks @ptoly, [your comment](https://github.com/react-icons/react-icons/pull/317#issuecomment-640561789) helped me a lot.

#### Why not replace the existing `Ionicons` icon set

The reason for keeping the existing `Ionicons (v4)`  icon set and adding `Ionicons v5` as a seperate icon set is to allow publishing the `react-icons` package with a minor version change and not breaking any existing code. _**The chance to break something is very high because `Ionicons v5` is a complete rewrite in terms of icon names.**_

Feel free to replace the existing `Ionicons` icon set with `Ionicons v5` if you publish v4 of the `react-icons` package.

#### Usage

```js
import { IoAddCircleOutline } from 'react-icons/io-5';
```

### CODE REVIEW

File: packages/react-icons/scripts/build.js [hunk 1]
```diff
@@ -180,8 +293,27 @@ async function writeIconModule(icon) {
...
+
+      let svgStr;
+
+      if (content.processWithSVGO) {
+        // console.log('RAW', svgStrRaw);
+        svgStr = await svgo.optimize(svgStrRaw).then(result => result.data);
+        // console.log('OPTIMIZED', svgStr);
+      } else {
+        svgStr = svgStrRaw;
+      }
```
Comment (phillystafford) [inline packages/react-icons/scripts/build.js:315]:
```suggestion
      const svgStr = content.processWithSVGO
        ? await svgo.optimize(svgStrRaw).then((result) => result.data)
        : svgStrRaw;
```

File: packages/react-icons/scripts/build.js [hunk 2]
```diff
@@ -180,8 +293,27 @@ async function writeIconModule(icon) {
     const entryDts = generateIconsEntry(icon.id, "dts");
     await appendFile(path.resolve(DIST, "all.d.ts"), entryDts, "utf8");
 
+    let count = 0;
```
Comment (phillystafford) [inline packages/react-icons/scripts/build.js:296]: I might be missing some context here but it looks like count is redundant

```suggestion
```

File: packages/react-icons/scripts/build.js [hunk 3]
```diff
@@ -180,8 +293,27 @@ async function writeIconModule(icon) {
...
+
     for (const file of files) {
-      const svgStr = await promisify(fs.readFile)(file, "utf8");
+      if (count > 3) {
+        // break;
+      }
+
+      count += 1;
```
Comment (phillystafford) [inline packages/react-icons/scripts/build.js:303]:
```suggestion
```

**Review by phillystafford — COMMENTED** [review]
Hi @florianstahr, I hope you don't mind me reviewing your PR but I'm keen to see the V5 Ionicons getting merged in too. If I can help out in any way please let me know 🙏

**Comment by CitizenBeta** [comment]
What's left to be done on this?

File: packages/react-icons/scripts/build.js (hunk 1 above)
Comment (florianstahr) [reply packages/react-icons/scripts/build.js]: I've applied the suggested changes. See 5a955e2.

File: packages/react-icons/scripts/build.js (hunk 2 above)
Comment (florianstahr) [reply packages/react-icons/scripts/build.js]: Yes, I've accidentally left that there. Removed in 5a955e2.

File: packages/react-icons/scripts/build.js (hunk 3 above)
Comment (florianstahr) [reply packages/react-icons/scripts/build.js]: Removed in 5a955e2.

**Comment by kamijin-fanta** [comment]
Thank you for your contribution!

I have committed to follow upstream and small changes such as renaming. Please use `react-icons/io5`.

**Pull request merged**